### PR TITLE
Problem #28: document._currentScript not needed

### PR DESCRIPTION
**Pull request by kegor**
First iteration for #28 fix.

### CODE REVIEW

**Comment by kegor** [comment]
@warpech, @tomalec I forgot to attach results of the local tests:
image
— this test environment a little new for me, is it looks ok?

File: examples/partials/concatenated.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	<script>
 		// document.currentScript.previousElementSibling.innerText = document.currentScript.model.path;
 		// Use polyfill of .currentScript if available
-		(document._currentScript || document.currentScript).previousElementSibling.innerText = (document._currentScript || document.currentScript).model.path;
+		document.currentScript.previousElementSibling.innerText = document.currentScript.model.path;
```
Comment (tomalec) [inline examples/partials/concatenated.html]: Could you remove the redundant comment from l.5-6

File: examples/partials/concatenated_scoped.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 		<script>
 			// document.currentScript.previousElementSibling.innerText = document.currentScript.model.data;
 			// Use polyfill of .currentScript if available
-			(document._currentScript || document.currentScript).previousElementSibling.innerText = (document._currentScript || document.currentScript).model.data;
+			document.currentScript.previousElementSibling.innerText = document.currentScript.model.data;
```
Comment (tomalec) [inline examples/partials/concatenated_scoped.html]: Could you remove comments above?

File: examples/partials/concatenated.html (hunk 1 above)
Comment (kegor) [reply examples/partials/concatenated.html]: Done.

File: examples/partials/concatenated_scoped.html (hunk 2 above)
Comment (kegor) [reply examples/partials/concatenated_scoped.html]: Done.

**Pull request merged**